### PR TITLE
Add contact  form

### PR DESCRIPTION
**Pull request by FilbertKimasa**
In this branch the following was done:

- Add contact form section
- Add footer section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -361,6 +361,82 @@ <h3 class="skill-title" id="skill-title-3">
...
+            maxlength="50"
+          ></textarea>
+          <input
+            type="submit"
+            name="submit-btn"
+            id="submit-btn"
+            value="Get in touch"
+            class="btn-get-in-touch"
+            required
+            maxlength="50"
```
Comment (Ibizugbe) [inline index.html]: Nice job so far😉. It is a requirement that the message field accepts up to 500 characters and not 50😉. Kindly fix this😉.
image

```suggestion
            maxlength="500"

```

File: style.css [hunk 2]
```diff
@@ -450,3 +450,139 @@
...
+.btn-get-in-touch:hover {
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.btn-get-in-touch:active {
+  background-color: #e05304;
+  color: #f4f1f0;
+}
```
Comment (Ibizugbe) [inline style.css]: Nice job styling your project to match the given design👍. But there are still some improvements we can make to make our codebase look professional. Currently, I noticed there are a lot of repetitive styles and this is not a good CSS practice as you ought to keep your code DRY. this helps you save some lines of code and make your codebase easily maintainable. for instance, all buttons have the same colors and border properties, both in hover, active and normal state, Hence you can just use one class to represent that in your styles and call it in each of the buttons 😉. In your case, you are repeating the same style in different classes and this is not a good `CSS` practice😉.

image

Kindly ensure your code is DRY, use one class for every similar style and call it in the respective HTML element😉.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @FilbertKimasa  @AbdulSalamAbro 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [ ] Linters are passing👍.
- [x] Github flow is used👍.
- [x] Project matches the design👍.
- [x] Button interactions have been implemented for each state👍.
- [x] Formspre service has been integrated in the from and everything works fine👍.
- [x] Layout is done with flexbox👍.
- [x] The correct type attribute has been added to all input fields👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Correct workflow ✔️ 
- HTML validation fixed ✔️ 
- DRY principle followed ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**